### ISSUE TITLE
How do I configure and test icesound?

### ISSUE THREAD

**Issue by cheapy** [bbidulock/icewm#334]
I'm trying 1.4.3.0~pre-20181030 from the testing repo on antiX 17.3.1 on a Dell Latitude E5540 Laptop.  Sound on the laptop is working via card: HDA Intel PCH according to AlsaMixer 1.1.3, and the test sound works, but when I try

icesound --list-unterfaces
AO,
icesound --list-files
startup.wav 
+++more
icesound --play=startup.wav 
no sound

Maybe its not configured to play?  I didn't find anything about how to configure or test it.

Thanks :)  IceWM is definitely my best window manager!

**Comment by gijsbers** [comment]
AO is a library wrapper over other sound interfaces.
You might need to configure it a bit and tell it which real interface it needs to use.
Does "man -k libao" give you something?
Figure out which AO configuration file you need to edit.
Then try something like: `.../icesound -i AO -v -p 2` where 2 is a sound number to play and -v gives extra info. What default sound interface(s) does antiX support?
Ideally there would be more than just "AO".

**Comment by cheapy** [comment]
Thanks, I think I got it except for the themes.  I found an old post where someone gave something close to the command below.  I got it working but didn't understand if there was a way to set a sound theme.  I use AlsaMixer, and therefore chose ALSA.

added ~/.icewm/sounds folder under regular user id, added folders underneath with sound themes.  It worked when I copied a set of wav files to the ~/.icewm/sounds folder

in ~/.icewm/startup will add:

icesound --interface=ALSA --sample-dir=~/.icewm/sounds &

I did try your command above but it get errors:

$ icesound -i AO -v -p 2
icesound: Warning: Unsupported interface: AO.
bobc@hpdv9917d:~/Downloads
$ icesound -i A -v -p 2
icesound: Warning: Unsupported interface: A.
bobc@hpdv9917d:~/Downloads
$ icesound -i O -v -p 2
icesound: Warning: Unsupported interface: O.
bobc@hpdv9917d:~/Downloads
$ icesound -V
icesound 1.4.2, Copyright 1997-2003 Marko Macek, 2001 Mathias Hasselmann.
